.config

```diff
@@ -59,7 +59,7 @@ CONFIG_EXTRA_LDLIBS=""
 #
 # Installation Options ("make install" behavior)
 #
-# CONFIG_INSTALL_APPLET_SYMLINKS is not set
+CONFIG_INSTALL_APPLET_SYMLINKS=y
```

Comment:
Symlinks are not needed for the container with the single process.

Reply:
Thanks @abitrolly I commented that one out then (and added the "is not set")